datanode/src/main/java/com/odysseusinc/arachne/datanode/dto/submission/SubmissionDTO.java

```diff
@@ -30,7 +31,7 @@ public class SubmissionDTO {
     private String analysis;
     private AnalysisOrigin origin;
     private DataSourceDTO dataSource;
-    private String status;
+    private AnalysisState state;
```

Comment:
Since this DTO is only used for rendering, I think we can avoid extra changes by keeping the name and String type here.